### PR TITLE
Add githubstudent's pin to the map.

### PR DESCRIPTION
**Pull request by KhanhDangDuy**
Adding Bach Khoa University to the map.

### CODE REVIEW

**Comment by Miari** [comment]
Add "---" and the and of the file and commit changes. After it hte check will be successful

**Comment by KhanhDangDuy** [comment]
I modified it. Could you check it again? 
Thanks!!

File: pins/githubstudent.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: githubstudent
```
Comment (crichID) [inline pins/githubstudent.json]: @KhanhDangDuy please replace `githubstudent` with your username!

**Comment by Miari** [comment]
Congratulations!

**Pull request merged**